include/envoy/router/route_config_provider_manager.h

```diff
@@ -25,6 +25,7 @@ namespace Router {
 class RouteConfigProviderManager {
 public:
   virtual ~RouteConfigProviderManager() = default;
+  using OptionalHttpFilters = std::set<std::string>;
```

Comment:
`absl::flat_hash_set`?

Reply:
yea, it can be absl::flat_hash_set, thanks!